### PR TITLE
Fix missing listPool

### PR DESCRIPTION
**Pull request by ZechariahTan**
* Add missing functionality to revert filtered pool list

### CODE REVIEW

**Review by markmcwong — COMMENTED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/ListPoolCommand.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_POOLS;
+
+import seedu.address.model.Model;
+
+/**
+ * Lists all passengers in the address book to the user.
```
Comment (chewterence) [inline src/main/java/seedu/address/logic/commands/ListPoolCommand.java]: did you mean list all pools?

File: src/test/java/seedu/address/logic/commands/ListPoolCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.pool.PooledPassengerContainsKeywordsPredicate;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for ListCommand.
```
Comment (chewterence) [inline src/test/java/seedu/address/logic/commands/ListPoolCommandTest.java]: did you mean ListPoolCommand

**Review by chewterence — CHANGES_REQUESTED** [review]
almost LGTM, only a few minor documentation errors, changes to be made.

**Review by JoelHo — APPROVED** [review]
LGTM

**Pull request merged**